godtools/Managers/LanguagesManager.swift

```diff
@@ -46,21 +46,16 @@ class LanguagesManager: GTDataManager {
         return loadFromDisk(id: GTSettings.shared.parallelLanguageId!)
     }
     
-    func loadFromDisk() -> [Language] {
-        languages = findAllEntities(Language.self)
-        
-        languages = languages.sorted { (language1, language2) -> Bool in
-            return language1.localizedName().compare(language2.localizedName()).rawValue < 0
-        }
-        
+    func loadFromDisk() -> List<Language> {
+        languages = findAllEntities(Language.self, sortedByKeyPath: "localizedName")
         return languages
     }
     
-    func loadFromRemote() -> Promise<[Language]> {
+    func loadFromRemote() -> Promise<List<Language>> {
```

Comment:
Same as before, we can set thinks easier to read if we use the `typealias` for creating something like `Languages`